packages/renderers-vixen-parser/e2e/raydium-amm-v4-parser/codama.cjs

```diff
@@ -0,0 +1,175 @@
+                'tradeFeeNumerator',
+                'tradeFeeDenominator',
+                'pnlNumerator',
+                'pnlDenominator',
+                'swapFeeNumerator',
+                'swapFeeDenominator',
+            ].map(fieldName => ({
+                kind: 'structFieldTypeNode',
+                name: fieldName,
+                docs: [],
+                type: { kind: 'numberTypeNode', format: 'u64', endian: 'le' },
+            })),
+        },
+    });
+
+    const accounts = node.program.accounts;
+    // Add `ammConfig` account (not included in idl)
+    // https://github.com/raydium-io/raydium-amm/blob/master/program/src/state.rs#L860-L871
+    const ammConfig = {
+        kind: 'accountNode',
+        name: 'ammConfig',
+        docs: [],
+        data: {
+            kind: 'structTypeNode',
+            fields: [
+                {
+                    kind: 'structFieldTypeNode',
+                    name: 'pnlOwner',
+                    docs: [],
+                    type: { kind: 'publicKeyTypeNode' },
+                },
+                {
+                    kind: 'structFieldTypeNode',
+                    name: 'cancelOwner',
+                    docs: [],
+                    type: { kind: 'publicKeyTypeNode' },
+                },
+                {
+                    kind: 'structFieldTypeNode',
+                    name: 'pending1',
+                    docs: [],
+                    type: {
+                        kind: 'arrayTypeNode',
+                        item: { kind: 'numberTypeNode', format: 'u64', endian: 'le' },
+                        count: { kind: 'fixedCountNode', value: 28 },
+                    },
+                },
+                {
+                    kind: 'structFieldTypeNode',
+                    name: 'pending2',
+                    docs: [],
+                    type: {
+                        kind: 'arrayTypeNode',
+                        item: { kind: 'numberTypeNode', format: 'u64', endian: 'le' },
+                        count: { kind: 'fixedCountNode', value: 31 },
+                    },
+                },
+                {
+                    kind: 'structFieldTypeNode',
+                    name: 'createPoolFee',
+                    docs: [],
+                    type: { kind: 'numberTypeNode', format: 'u64', endian: 'le' },
+                },
+            ],
+        },
+        discriminators: [],
+    };
+    accounts.push(ammConfig);
+
+    const ammConfigIndex = definedTypes.findIndex(definedType => definedType.name === 'ammConfig');
+    definedTypes.splice(ammConfigIndex, 1);
+    const feesIndex = accounts.findIndex(account => account.name === 'fees');
+    accounts.splice(feesIndex, 1);
+
+    const updatedNode = { ...node, program: { ...node.program, definedTypes, accounts } };
+
+    visit(
+        updatedNode,
+        renderVisitor({
+            projectFolder: __dirname,
+            projectName: project,
+        }),
+    );
+
```

Comment:
@fernandodeluret that is a nice looking top level API. Thank you for making these changes.